deploy/docker/seqr/entrypoint.sh

```diff
@@ -75,26 +75,5 @@ python -u manage.py check
 # launch django server in background
 /usr/local/bin/start_server.sh
 
-if [ "$RUN_CRON_JOBS" ]; then
```

Comment:
`RUN_CRON_JOBS` has been off in production for a bit

Reply:
we shouldk probably fully remove it from the helm charts as well after this is deployed